src/CommunityToolkit.Maui.MediaElement/Handlers/MediaElementHandler.macios.cs

```diff
@@ -10,9 +11,29 @@ public partial class MediaElementHandler : ViewHandler<MediaElement, MauiMediaEl
 	/// <exception cref="NullReferenceException">Thrown if <see cref="MauiContext"/> is <see langword="null"/>.</exception>
 	protected override MauiMediaElement CreatePlatformView()
 	{
-		mediaManager ??= new(MauiContext ?? throw new InvalidOperationException($"{nameof(MauiContext)} cannot be null"), VirtualView);
+		if (MauiContext is null)
+		{
+			throw new InvalidOperationException($"{nameof(MauiContext)} cannot be null");
+		}
+
+		mediaManager ??= new(MauiContext, VirtualView);
+
+		// Retrieve the parenting page so we can provide that to the platform control
+		var parent = VirtualView.Parent;
+		while (parent != null)
```

Comment:
Wow, that was close!
```suggestion
		while (parent is not null)
```

Reply:
Old habits die hard...